README.md

```diff
@@ -1,6 +1,7 @@
 # Community Solid Server
-
+<!-- markdownlint-disable -->
 <img src="https://raw.githubusercontent.com/CommunitySolidServer/CommunitySolidServer/main/templates/images/solid.svg" alt="[Solid logo]" height="150" align="right"/>
+<!-- markdownlint-enable -->
```

Comment:
This line also violates line length... splitting the URL breaks it. Do we want to allow html otherwise? I thought this was an exception.

Reply:
Would still allow HTML tags. Can be useful in some cases and we might want to add additional images at some point in the future.

Reply:
Splitting the line right after the URL seems to be sufficient to not go over the character limit so can we do that instead of doing the `markdownlint-disable` then?

Would be nice if we could tell the linter to ignore line length in case there is an URL but there doesn't seem to be an option for that.